widget/embedded/src/hooks/useIframe/useIframe.ts

```diff
@@ -0,0 +1,55 @@
+import type { Messages } from '@lingui/core';
+
+import { useRef } from 'react';
+
+import { useAppStore } from '../../store/AppStore';
+
+import { isAppLoadedIntoIframe } from './useIframe.helpers';
+
+interface UseIframe {
+  send: (message: Messages) => void;
+  connectHeightObserver: (element: HTMLElement) => void;
+  disconnectHeightObserver: () => void;
+}
+
+function useIframe(): UseIframe {
+  const heightObserver = useRef<any>(null);
```

Comment:
you can remove any type and use "ResizeObserver | null"

Reply:
I'm tired of this. I tried first and it doesn't work, then I tried it right now it's working. sigh...
fixed.